pkg/manifest/manifest.go

```diff
@@ -5,38 +5,36 @@
 package manifest
 
 import (
+	"encoding"
 	"errors"
 	"net/http"
 
 	"github.com/ethersphere/bee/pkg/swarm"
 )
 
+// ErrNotFound is returned when an Entry is not found in the manifest
 var ErrNotFound = errors.New("manifest: not found")
 
-// Parser for manifest
-type Parser interface {
-	// Parse parses the encoded manifest data and returns the result
-	Parse(bytes []byte) (Interface, error)
-}
-
 // Interface for operations with manifest
 type Interface interface {
-	// Add a manifest entry to specified path
+	// Add a manifest entry to the specified path
 	Add(string, Entry)
-	// Remove reference from file on specified path
+	// Remove a manifest entry on the specified path
 	Remove(string)
-	// FindEntry returns manifest entry if one is found on specified path
+	// FindEntry returns a manifest entry if one is found in the specified path
 	FindEntry(string) (Entry, error)
```

Comment:
as with my comment below on `Get` prefixes, I think that we could live with just `Entry(path string)` here

Reply:
done